pandas/io/parsers/arrow_parser_wrapper.py

```diff
@@ -158,9 +160,29 @@ def read(self) -> DataFrame:
             parse_options=pyarrow_csv.ParseOptions(**self.parse_options),
             convert_options=pyarrow_csv.ConvertOptions(**self.convert_options),
         )
-        if self.kwds["dtype_backend"] == "pyarrow":
+
+        dtype_backend = self.kwds["dtype_backend"]
+
+        # Convert all pa.null() cols -> float64 (non nullable)
+        # else Int64 (nullable case)
+        # TODO: There has to be a better way... right?
+        if dtype_backend != "pyarrow":
+            new_schema = table.schema
+            if dtype_backend == "numpy_nullable":
+                new_type = pa.int64()
```

Comment:
Can't you just path the ``_arrow_dtype_mapping``?

Reply:
Will need to wait for #52223 then.

Reply:
LGTM, but yeah since https://github.com/pandas-dev/pandas/pull/52223 is close lets wait for that one then merge this